.circleci/config.yml

```diff
@@ -392,6 +394,15 @@ jobs:
             wget https://github.com/CraneStation/wasmtime/releases/download/dev/wasmtime-dev-x86_64-linux.tar.xz
             tar -xf wasmtime-dev-x86_64-linux.tar.xz
             cp wasmtime-dev-x86_64-linux/wasmtime ~/vms
+      - run:
+          name: get v8
+          command: |
+            wget https://nodejs.org/dist/v12.13.0/node-v12.13.0-linux-x64.tar.xz
```

Comment:
This seems a little odd.   We already have node installed via emsdk don't we.  Why download another copy?

Reply:
Oh good point, I thought we had only node there and not npm, but you're right, the emsdk provides that too...

Anyhow, it sounds like we may remove this and revert back to v8 in the emsdk download anyhow, if that roll sticks, so I'll leave this as is for the moment I think.